Foundation/NSPropertyList.swift

```diff
@@ -113,6 +113,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return data
     } else if let number = input as? NSNumber {
         return number
+    } else if let url = input as? NSURL {
```

Comment:
`NSBundle.mainBundle().infoDictionary!["CFBundleInfoPlistURL"]` returns this.

Looks like CoreFoundation adds this key to the actual plist contents here: https://github.com/apple/swift-corelibs-foundation/blob/4a6d90f0406c34e896bfd3e89585717496bb2c1f/CoreFoundation/PlugIn.subproj/CFBundle_InfoPlist.c#L534